pkg/investigations/ccam/ccam.go

```diff
@@ -86,33 +73,11 @@ func RemoveLimitedSupport(cluster *v1.Cluster, ocmClient ocm.Client, pdClient pa
 		return err
 	})
 	if err != nil {
-		logging.Errorf("Failed to remove CCAM Limited support reason from cluster. Attempting to alert Primary.")
-		originalErr := err
-		err := utils.WithRetries(func() error {
-			return pdClient.CreateNewAlert(buildAlertForCCAM(originalErr, cluster.ID()), pdClient.GetServiceID())
-		})
-		if err != nil {
-			logging.Errorf("Failed to alert Primary")
-			return err
-		}
-		logging.Info("Primary has been alerted")
-		return nil
+		logging.Errorf("Failed to remove CCAM Limited support reason from cluster.")
+		return err
 	}
 	if removedReason {
 		metrics.Inc(metrics.LimitedSupportLifted, alertType, pdClient.GetEventType(), ccamLimitedSupport.Summary)
 	}
 	return nil
 }
-
-// buildAlertForCCAM will return a NewAlert populated with cluster id and the specific error
-func buildAlertForCCAM(lsError error, clusterID string) pagerduty.NewAlert {
```

Comment:
Paging primary with this might be more confusing than just letting it reach primary the normal way. This basically never happens, and when it does it would just cause confusion as to what has and what hasn't been done here.

Reply:
i think we need to explicitly think about ccam as well. We mainly came from a perspective of removing LS automation for chgm. If we keep posting LS on ccam, my feeling is we need at least some possibility of removing it automatically.

Reply:
We have https://issues.redhat.com/browse/SDE-2424 for cyclic CCAM setting and removals with EDA. I think this is the cleaner streamlined way. 

The reason I want to keep LS for CCAM for now is that it often isn't the root to the issue resulting in a page. Sending a SL without telling the customer their cluster is not being supported doesn't match our action of sending and muting the alert.

Reply:
Agreed that it needs to be LS and pulled out of chgm logic. My concern was with having no automatic removal anymore, increasing the load for manual removal. But after this change we will have different behavior anyway: Not checking the aws account on resolving chgms -> we cannot tell if we have access -> no automatic removal possible.

Reply:
I had checked this with a pd query, and over the span of 4 weeks this resulted in only 3 cases where we would maybe want to remove limited support manually.

Taking only incidents in the past month, everything that had nodes shutdown for longer than that has certs expired and should stay in LS: 
```
pd incident:list --services=prod-deadmanssnitch --since=11/1/2023 --notes | grep -c "Your cluster is no longer checking in with Red Hat OpenShift Cluster Manager. Possible causes include stopped instances or a networking misconfiguration. If you have stopped the cluster instances, please start them again - stopping instances is not supported. If you intended to terminate this cluster then please delete the cluster in the Red Hat console"
Getting incidents 2/2 👍, 0/2 👎... done
Getting notes for 34 incidents 34/34 👍, 0/34 👎... done

3
```